### PR TITLE
CLOUD-165: Make liveness and readiness probes configurable

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | failed |
| storage | failed |
| demand-backup | failed |
| monitoring | failed |
| limits | failed |
| scaling | failed |
| scheduled-backup | failed |
| monitoring-2-0 | failed |
| arbiter | failed |
| upgrade | failed |
| service-per-pod | failed |
| operator-self-healing | passed |
| upgrade-consistency | failed |
| one-pod | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| self-healing | passed |
| storage | passed |
| demand-backup | failed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | failed |
| scaling | passed |
| monitoring-2-0 | passed |
| upgrade | failed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | passed |
| storage | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| scaling | passed |
| upgrade | failed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| arbiter | passed |
| upgrade-consistency | passed |
| service-per-pod | failed |
| one-pod | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | failed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| service-per-pod | failed |
| demand-backup | failed |
| scheduled-backup | failed |
| security-context | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| storage | passed |
| init-deploy | passed |
| demand-backup | failed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | failed |
| scaling | passed |
| monitoring-2-0 | failed |
| upgrade | failed |
| operator-self-healing | passed |
| arbiter | failed |
| upgrade-consistency | passed |
| service-per-pod | passed |
| one-pod | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| monitoring | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| demand-backup | failed |
| scheduled-backup | failed |
| arbiter | failed |
| service-per-pod | passed |
| security-context | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| demand-backup | failed |
| scheduled-backup | failed |
| service-per-pod | passed |
| security-context | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | failed |
| demand-backup | failed |
| scheduled-backup | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | failed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| demand-backup | failed |
| scheduled-backup | failed |
| upgrade | failed |
| upgrade-consistency | passed |

File: e2e-tests/build-and-run [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
 test_dir="$(dirname $0)"
 . $(dirname $0)/functions
 src_dir="$(realpath $test_dir/..)"
 
-$test_dir/build
+if [[ $nobuild -eq 0  ]]; then
+    echo "building, use -nb or --nobuild to suppress this"
+    $test_dir/build || exit
+fi
+
```
Comment (delgod) [inline e2e-tests/build-and-run]: could we unify approach for both operators, in PXC we do
```
BUILD=0 ./e2e-tests/build-and-run
```
code - https://github.com/percona/percona-xtradb-cluster-operator/blob/master/e2e-tests/build#L45-L47

File: e2e-tests/scheduled-backup/compare/statefulset_some-name-backup-coordinator-oc.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@ spec:
         livenessProbe:
           failureThreshold: 3
           initialDelaySeconds: 5
-          periodSeconds: 10
+          periodSeconds: 30
```
Comment (delgod) [inline e2e-tests/scheduled-backup/compare/statefulset_some-name-backup-coordinator-oc.yml]: is it really changed?

File: e2e-tests/demand-backup/compare/statefulset_some-name-backup-coordinator-oc.yml [hunk 3]
```diff
@@ -60,7 +60,7 @@ spec:
         livenessProbe:
           failureThreshold: 3
           initialDelaySeconds: 5
-          periodSeconds: 10
+          periodSeconds: 30
```
Comment (delgod) [inline e2e-tests/demand-backup/compare/statefulset_some-name-backup-coordinator-oc.yml]: is it really changed?

File: pkg/apis/psmdb/v1/psmdb_defaults.go [hunk 4]
```diff
@@ -126,14 +126,52 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
...
-			replset.LivenessInitialDelaySeconds = &ld
+		if replset.LivenessProbe == nil {
+			replset.LivenessProbe = &corev1.Probe{
+				Handler: corev1.Handler{
+					Exec: &corev1.ExecAction{
+						Command: []string{
+							"mongodb-healthcheck",
+							"k8s",
+							"liveness",
+						},
```
Comment (delgod) [inline pkg/apis/psmdb/v1/psmdb_defaults.go]:
```suggestion
						        "--startupDelaySeconds",
						        startupDelaySeconds
						},
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| self-healing | passed |
| init-deploy | failed |
| demand-backup | failed |
| monitoring | failed |
| limits | failed |
| scaling | passed |
| scheduled-backup | failed |
| monitoring-2-0 | failed |
| upgrade | failed |
| arbiter | failed |
| service-per-pod | failed |
| security-context | failed |
| operator-self-healing | passed |
| upgrade-consistency | failed |
| one-pod | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| init-deploy | passed |
| self-healing | passed |
| demand-backup | passed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| operator-self-healing | passed |
| arbiter | passed |
| upgrade | failed |
| one-pod | passed |
| service-per-pod | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| storage | failed |
| init-deploy | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| scaling | passed |
| upgrade | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | passed |
| operator-self-healing | passed |
| limits | passed |
| one-pod | failed |
| scaling | passed |
| storage | passed |
| demand-backup | failed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| scheduled-backup | failed |
| security-context | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| monitoring | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| one-pod | failed |
| arbiter | passed |
| service-per-pod | failed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| demand-backup | failed |
| security-context | failed |
| scheduled-backup | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| demand-backup | failed |
| security-context | failed |
| scheduled-backup | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| self-healing | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | failed |
| upgrade | failed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | failed |
| security-context | failed |

File: e2e-tests/functions [hunk 5]
```diff
@@ -8,7 +8,7 @@ GIT_BRANCH=${VERSION:-$(git rev-parse --abbrev-ref HEAD | sed -e 's^/^-^g; s^[.]
 API="psmdb.percona.com/v1-4-0"
 IMAGE=${IMAGE:-"perconalab/percona-server-mongodb-operator:${GIT_BRANCH}"}
 IMAGE_PMM=${IMAGE_PMM:-"perconalab/percona-server-mongodb-operator:master-pmm"}
-IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:master-mongod4.2"}
+IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:master-mongod4.0"}
```
Comment (delgod) [inline e2e-tests/functions]:
```suggestion
IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:master-mongod4.2"}
```

File: e2e-tests/build-and-run (hunk 1 above)
Comment (blez) [reply e2e-tests/build-and-run]: I like `-nb` flag much more than setting `BUILD` env variable, but if you insist I can change it.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| self-healing | passed |
| demand-backup | failed |
| storage | failed |
| scheduled-backup | failed |
| limits | passed |
| monitoring | passed |
| scaling | passed |
| upgrade | failed |
| operator-self-healing | failed |
| monitoring-2-0 | failed |
| upgrade-consistency | passed |
| one-pod | failed |
| arbiter | failed |
| service-per-pod | passed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| self-healing | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| scaling | passed |
| upgrade | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | failed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | passed |
| demand-backup | failed |
| scheduled-backup | failed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| self-healing | passed |
| demand-backup | failed |
| init-deploy | passed |
| monitoring | passed |
| scheduled-backup | passed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | failed |
| upgrade | failed |
| operator-self-healing | passed |
| arbiter | passed |
| upgrade-consistency | passed |
| one-pod | passed |
| service-per-pod | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| init-deploy | passed |
| limits | passed |
| scaling | failed |
| demand-backup | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | passed |
| scheduled-backup | passed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | failed |
| storage | passed |
| init-deploy | passed |
| demand-backup | passed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | passed |
| scaling | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| upgrade | passed |
| arbiter | passed |
| one-pod | passed |
| upgrade-consistency | failed |
| service-per-pod | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| self-healing | passed |
| storage | passed |
| demand-backup | passed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | passed |
| scaling | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| upgrade | passed |
| arbiter | passed |
| one-pod | passed |
| upgrade-consistency | failed |
| service-per-pod | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| self-healing | passed |
| demand-backup | passed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | passed |
| scaling | passed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| arbiter | passed |
| upgrade | passed |
| one-pod | passed |
| service-per-pod | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-300-bf3f5bf6

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| arbiter | passed |
| service-per-pod | passed |
| security-context | passed |

File: e2e-tests/liveness/conf/liveness-rs0.yml [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        resources:
+          requests:
+            storage: 1Gi
+    size: 3
+    containerSecurityContext:
+      privileged: true
+    podSecurityContext:
+      fsGroup: 1001
+      supplementalGroups: [1001, 1002, 1003]
+    livenessProbe:
```
Comment (delgod) [inline e2e-tests/liveness/conf/liveness-rs0.yml]: could we check also readiness probe?

File: e2e-tests/liveness/conf/liveness-rs0.yml [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+      failureThreshold: 5
+      initialDelaySeconds: 61
+      periodSeconds: 31
+      successThreshold: 1
```
Comment (delgod) [inline e2e-tests/liveness/conf/liveness-rs0.yml]: could we also increase successThreshold in test?
```suggestion
      successThreshold: 2
```

Comment (blez) [reply e2e-tests/liveness/conf/liveness-rs0.yml]: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
Based on k8s docs successThreshold `Must be 1 for liveness`

File: e2e-tests/functions [hunk 8]
```diff
@@ -9,7 +9,7 @@ API="psmdb.percona.com/v1-4-0"
 IMAGE=${IMAGE:-"perconalab/percona-server-mongodb-operator:${GIT_BRANCH}"}
 IMAGE_PMM=${IMAGE_PMM:-"perconalab/percona-server-mongodb-operator:master-pmm"}
 IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:master-mongod4.2"}
-IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-server-mongodb-operator:master-backup"}
+IMAGE_BACKUP=${IMAGE_BACKUP:-"percona/percona-server-mongodb-operator:1.2.0-backup"}
```
[the remark this answers is not in the document]
Comment (blez) [reply e2e-tests/functions]: Should be revert back?

[the remark this answers is not in the document]
Comment (delgod) [reply e2e-tests/functions]: yes, please

File: e2e-tests/liveness/conf/liveness-rs0.yml (hunk 7 above)
Comment (delgod) [reply e2e-tests/liveness/conf/liveness-rs0.yml]: OK, let's try to change it for readiness

File: e2e-tests/build-and-run (hunk 1 above)
Comment (delgod) [reply e2e-tests/build-and-run]: I am ok, on any approach, but please let it be identical in both products

File: e2e-tests/liveness/conf/liveness-rs0.yml (hunk 6 above)
Comment (blez) [reply e2e-tests/liveness/conf/liveness-rs0.yml]: Done https://github.com/percona/percona-server-mongodb-operator/pull/300/commits/bfde9c0cd0f7b99b8a87eec781a4360e9510cb76

File: e2e-tests/functions (hunk 8 above)
[the remark this answers is not in the document]
Comment (blez) [reply e2e-tests/functions]: Done https://github.com/percona/percona-server-mongodb-operator/pull/300/commits/44f7d59b2079c222dcabc1f7cac46e97a6f8ffd5

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| self-healing | passed |
| storage | passed |
| demand-backup | passed |
| monitoring | passed |
| limits | passed |
| scheduled-backup | passed |
| monitoring-2-0 | failed |
| scaling | passed |
| operator-self-healing | failed |
| arbiter | passed |
| upgrade | passed |
| one-pod | passed |
| service-per-pod | failed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| init-deploy | passed |
| demand-backup | failed |
| monitoring | passed |
| scheduled-backup | failed |
| limits | passed |
| monitoring-2-0 | passed |
| upgrade | passed |
| scaling | passed |
| operator-self-healing | passed |
| arbiter | failed |
| one-pod | failed |
| service-per-pod | failed |
| security-context | failed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| self-healing | passed |
| limits | passed |
| operator-self-healing | passed |
| one-pod | passed |
| scaling | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| demand-backup | passed |
| service-per-pod | failed |
| security-context | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| monitoring | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| storage | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | failed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |

File: e2e-tests/build-and-run (hunk 1 above)
Comment (blez) [reply e2e-tests/build-and-run]: https://github.com/percona/percona-server-mongodb-operator/pull/300/commits/b26bb9b5c7917ece560211316a039dbc0747f297

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| self-healing | passed |
| storage | passed |
| demand-backup | failed |
| limits | failed |
| monitoring | passed |
| scaling | failed |
| scheduled-backup | failed |
| upgrade | failed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| arbiter | passed |
| service-per-pod | failed |
| one-pod | failed |
| security-context | failed |
| upgrade-consistency | failed |

**Pull request merged**